src/Workspaces/Core/Portable/Microsoft.CodeAnalysis.Workspaces.csproj

```diff
@@ -309,8 +309,12 @@
     <InternalsVisibleTo Include="Microsoft.VisualStudio.LanguageServices.TypeScript" Key="$(TypeScriptKey)" />
     <InternalsVisibleTo Include="Roslyn.Services.Editor.TypeScript.UnitTests" Key="$(TypeScriptKey)" />
     <InternalsVisibleTo Include="Microsoft.VisualStudio.LanguageServices.Remote" Key="$(RemoteLanguageServiceKey)" />
+    <InternalsVisibleTo Include="Microsoft.VisualStudio.LanguageServices.Remote.15.7" Key="$(RemoteLanguageServiceKey)" />
+    <InternalsVisibleTo Include="Microsoft.VisualStudio.LanguageServices.Remote.15.8" Key="$(RemoteLanguageServiceKey)" />
```

Comment:
Odd to me this is coming back -- do we need to do a merge with -vs-deps?

Reply:
@jinujoseph, @jasonmalinowski, let us agree which branch to merge master or master-vs-deps? 

---
In reply to: [236132130](https://github.com/dotnet/roslyn/pull/29016#discussion_r236132130) [](ancestors = 236132130)

Reply:
Oh definitely master-vs-deps; you might want to merge/rebase though to make those things go away from your diff, since they've also made it into your target.